draft-ietf-ppm-dap.md

```diff
@@ -810,13 +810,22 @@ as specified by the VDAF. It then encrypts each input share as follows:
 ~~~
 enc, payload = SealBase(pk,
   "dap-02 input share" || 0x01 || server_role,
-  task_id || metadata || public_share, input_share)
+  input_share_aad, input_share)
 ~~~
 
 where `pk` is the aggregator's public key; `server_role` is the Role of the
-intended recipient (`0x02` for the leader and `0x03` for the helper), `task_id`
-is the task ID, `metadata` is the report metadata, and `input_share` is the
-Aggregator's input share.
+intended recipient (`0x02` for the leader and `0x03` for the helper),
+`input_share` is the Aggregator's input share, and `input_share_aad` is an
+encoded message of type `InputShareAAD`, constructed from the same values as
+the corresponding fields in the report:
+
+~~~
+struct {
+  TaskID task_id;
+  ReportMetadata metadata;
+  opaque public_share<0..2^32-1>;
+} InputShareAAD;
```

Comment:
nit: if we follow he same camel case convention of types, this should be `InputShareAad`. Same applies to `AggregateShareAad`